tests/pages/DataOverlayPageObject.ts

```diff
@@ -0,0 +1,33 @@
+import { Page } from '@playwright/test';
+
+export class DataOverlayPageObject {
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  get button() {
```

Comment:
This is weird to have button always get the first dataOverlayMenu. Is it being used? If so rename to make it clear that it is a specific (i.e. the first) instance. If not please remove.